README.rdoc

```diff
@@ -1,13 +1,17 @@
 = ValidatesTimeliness {<img src="https://travis-ci.org/adzap/validates_timeliness.svg?branch=master" alt="Build Status" />}[https://travis-ci.org/adzap/validates_timeliness]
 
-* Source:  http://github.com/adzap/validates_timeliness
-* Issues:  http://github.com/adzap/validates_timeliness/issues
+* Source:  https://github.com/adzap/validates_timeliness
+* Issues:  https://github.com/adzap/validates_timeliness/issues
 
 == Description
 
-Complete validation of dates, times and datetimes for Rails 5.x and ActiveModel.
+Complete validation of dates, times and datetimes for Rails 6.x and ActiveModel.
 
-If you a looking for the old version for Rails 4.x go here [https://github.com/adzap/validates_timeliness/tree/4-0-stable].
+Old Rails versions:
+
+* Rails 4.x: [https://github.com/adzap/validates_timeliness/tree/4-0-stable]
+
+* Rails 5.x: [https://github.com/adzap/validates_timeliness/tree/5-0-stable]
```

Comment:
opinionated, feel free to suggest a better approach

Reply:
this is food